plugins/scala_test/tests/resources/input/sample.scala

```diff
@@ -0,0 +1,7 @@
+package com.scala.piranha
+
+import com.uber.michelangelo.AbstractSparkSuite
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.types.{DoubleType, StringType, StructField, StructType}
+import org.scalatest.{BeforeAndAfter, Matchers}
+import org.scalatest.mock.MockitoSugar
```

Comment:
Shouldn't we have a case where an import like `import pkg1.pkg2.{A, B}` is replaced as `import pkg1.pkg2.{C, B}` and also the simple `import pkg3.pkg4.D` -> `import pkg5.pkg6.E` case?

Reply:
hmmm. Actually the solution u suggest looks clean when the before and after type have a significant overlap in their qualified name. Else we have to "infer" the level to split the type name. 

From: `a.b.c.D` to `a.e.f.g.H`.  
Before: `import a.b.c.{D, E}` 
After: `import a.{b.c.E, e.f.H}`
or add some extra logic to heuristically decide when it is not a good idea to add a nested import, and add a simple import at those times.
-------------

From: `a.b.c.D` to `a.b.c.H`.  
Before: `import a.b.c.{D, E}` 
After: `import a.b.c.{H, E}`

-------
( actually @raviagarwal7 suggested adding simple import and keep the rewrite logic less complicated. We believe that we can use Scala linters to re-org the imports like we want to)

Reply:
Not sure I understand this. I am not saying the rewrite done in these tests is incorrect, but I think there are missing cases given the rewrite rules you added. Specifically the `update_simple_import_{...}` rules. My question here is about test coverage for the rules/logic added.